tutorials/rendering/viewports.rst

```diff
@@ -130,6 +101,45 @@ is possible to do so by setting :ref:`world_2d<class_Viewport_property_world_2d>
 
 For an example of how this works, see the demo projects `3D in 2D <https://github.com/godotengine/godot-demo-projects/tree/master/viewport/3d_in_2d>`_ and `2D in 3D <https://github.com/godotengine/godot-demo-projects/tree/master/viewport/2d_in_3d>`_ respectively.
 
+Cameras (2D & 3D)
+-----------------
+
+When using a :ref:`Camera3D <class_Camera3D>` or
+:ref:`Camera2D <class_Camera2D>`, it will always display on the
+closest parent :ref:`Viewport <class_Viewport>` (going towards the root), which has an assigned World3D.
+That is, it will display on either the root viewport, or the closest parent viewport that either has 
+:ref:`Own World3D <class_Viewport_property_own_world_3d>` set to true, or a value in its 
+:ref:`World3D <class_World3D>`.
+
+For example, if the :ref:`SubViewport <class_SubViewport>` has an assigned :ref:`World3D <class_World3D>`, then in the following hierarchy:
+
+.. image:: img/cameras.webp
+
+``CameraA`` will display on the Root :ref:`Viewport <class_Viewport>` and it will draw ``MeshA``. ``CameraB``
+will be captured by the :ref:`SubViewport <class_SubViewport>` along with ``MeshB``. Even though ``MeshB`` is in the scene
+hierarchy, it will still not be drawn to the Root Viewport. Similarly, ``MeshA`` will not
+be visible from the SubViewport because SubViewports and their :ref:`World3D <class_World3D>` only
+capture nodes which are their children in the graph.
+
+.. note::
+
+    This behavior only occurs when the SubViewport has its own :ref:`World3D <class_World3D>` to guide its rendering. Otherwise, ``MeshB`` will continue up the graph, and be rendered on the first viewport which does have an assigned :ref:`World3D <class_World3D>`; in this case, the Root :ref:`Viewport <class_Viewport>`.
```

Comment:
```suggestion
    This behavior only occurs when the SubViewport has its own :ref:`World3D <class_World3D>` to guide its rendering. Otherwise, ``MeshB`` will continue up the graph,
    and be rendered on the first viewport which does have an assigned :ref:`World3D <class_World3D>`; in this case, the Root :ref:`Viewport <class_Viewport>`.
```
Keep the lines around the same length